mpqp/core/instruction/measurement/basis.py

```diff
@@ -190,13 +258,18 @@ class HadamardBasis(VariableSizeBasis):
     """
```

Comment:
same here: I would put an example showcasing the symbols for Hadamard (-+)

Reply:
Done.